jfltexx.json

```diff
@@ -0,0 +1,24 @@
+{
+  "response": [
+    {
+        "maintainer": "Santiago García Mantiñán (manty)",
+        "oem": "Samsung",
+        "device": "Galaxy S4 (jfltexx)",
```

Comment:
device codename is not needed here as it's already set by device json file

Reply:
Ok, I used the two typical model names in short form this time ;-)